guides/source/active_record_migrations.md

```diff
@@ -365,9 +365,7 @@ create_join_table :products, :categories
 ```
 
 which creates a `categories_products` table with two columns called
-`category_id` and `product_id`. These columns have the option `:null` set to
-`false` by default. This can be overridden by specifying the `:column_options`
-option.
+`category_id` and `product_id`. (Note that the table's name is defined by the lexical order of the class names - and not by the order of the parameters.) These columns have the option `:null` set to `false` by default. This can be overridden by specifying the `:column_options` option.
 
```

Comment:
It might be good to wrap lines at 80 chars like the original formatting and as per the contributing guidelines.